Source/CarthageKit/Xcode.swift

```diff
@@ -6,6 +6,8 @@ import ReactiveSwift
 import ReactiveTask
 import XCDBLD
 
+import struct Utility.Version
```

Comment:
You'll need to merge `master` because I merged #2720.

Reply:
Will do.